components/PlantCard.js

```diff
@@ -0,0 +1,46 @@
+/* Was macht diese Seite ?
+Die component visualisiert eine card:
+- PlantName: Aloe Vera
+- PlantBotanicalname: Aloe barbadensis miller
+- Image mit imageUrl
+- Button für "see more"
+ */
+
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function PlantCard({ plant }) {
+  return (
+    <Container>
+      <h2>{plant.name}</h2>
+      <StyledParagraph>{plant.botanicalName}</StyledParagraph>
+      <RoundImage alt="image" src={plant.imageUrl}></RoundImage>
```

Comment:
I think we need to make the alt more descriptive for accessibility reasons. 

Something like

``` javascript
  alt={`Photo of ${plant.name}`} 
```

Reply:
that makes sense. thank you